c_common/models/system_models/src/extra_monitor_support.c

```diff
@@ -609,25 +530,49 @@ static ushort my_addr;
 //! The SARK virtual processor information table in SRAM.
 static vcpu_t *const _sark_virtual_processor_info = (vcpu_t *) SV_VCPU;
 
-//! \brief DSG metadata
+//! The magic number that marks a valid DSE metadata descriptor
+#define DSE_MAGIC       0xAD130AD6
```

Comment:
Yes, it most certainly is, but the data spec implementation uses spin1_api (for logging) and the extra monitor doesn't (because it needs low-level interrupt access).

Reply:
having reviewed this again. i laded on this AGAIN, and i still think (and @rowleya also thinks) that we shouldn't be bringing this in as a clone. we should adjust the dsg code accordingly. either by not using logs in it, or having 2 version which have the different types of print so you can import the right one. ill leave the other pointers of this block resolved, but the overarching issue still needs thought.